### PR TITLE
Commit chess tiles made by razetime to master(GCI Task)

### PR DESCRIPTION
**Pull request by razetime**
Chess tiles for labyrinth made and committed to master. tiles.js has been modified to ddeclare the state and behaviour of these tiles.

I modified the labyrinth, you can test my changes here:
http://rawgit.com/razetime/labyrinth/master/index.html

I have made some new chess tiles for the labyrinth which display messages when you happen upon them. I have added 2 walls, one floor tile, and 2 doors.
This was made for the Labyrinth: Create tiles for a landscape task at Google Code-In.
https://codein.withgoogle.com/dashboard/task-instances/5274354759761920/

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ ImageCollection.prototype.showIn = function(container) {
 ImageCollection.prototype.playerEnters = function(player) {
   this.container.classList.add("current");
 };
-ImageCollection.prototype.playerLeaves = function(player) {
+ImageCollection.prototype.playerLeaves = function(player) {top (copy)
```
Comment (niccokunzmann) [inline js/tiles.js]: When I want to play your labyrinth, I get an error: Reference Error copy is not defined.

Could you check that the labyrinth is still playable?

**Comment by niccokunzmann** [comment]
Awesome! Three new tiles. Would you like to put them into the levels.js file, so we can reach them?

**Comment by niccokunzmann** [comment]
Thanks for adding this link: http://rawgit.com/razetime/labyrinth/master/index.html as it made it easy for me to review.

**Comment by niccokunzmann** [comment]
> Write clear meaningful git commit messages (Do read http://chris.beams.io/posts/git-commit/)

https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md

The game is playable now. If you like to submit the task, please add the tiles to the labyrinth so we can use them.

**Comment by razetime** [comment]
As I'm not sure of how to use the levels file, I have added my tiles to the levels file in a comment.

**Pull request merged**